vivisect/__init__.py

```diff
@@ -182,6 +182,8 @@ def __init__(self, **kwargs):
         self.addVaSet('EmucodeFunctions', (('va', VASET_ADDRESS),))
         self.addVaSet('FuncWrappers', (('va', VASET_ADDRESS), ('wrapped_va', VASET_ADDRESS),))
         self.addVaSet('thunk_reg', ( ('fva', VASET_ADDRESS), ('reg', VASET_STRING), ('tgtval', VASET_INTEGER)) )
+        self.addVaSet('ResolvedImports', (('va',VASET_ADDRESS), ('symbol', VASET_STRING), \
```

Comment:
You don't need the trailing `\` here because you are in an open paren ( pet peeve about line breaks ;)

Reply:
:+1: